### PR TITLE
Fix/10063 various collectibles bugs

### PR DESCRIPTION
**Pull request by dlipicar**
### What does the PR do

Fixes https://github.com/status-im/status-desktop/issues/10063

Fix several issues related to collectibles:

- Properly handle errors when fetching collectibles. There's now an `Error` state, which prevents further fetching until a reset occurs.
- Added missing init, fixing periodic re-fetches
- Reduced chunk size to improve responsiveness
- Fixed loading items on initial fetch
- Made collectibles screen match the design when an error occurs

### CODE REVIEW

**Review by Khushboo-dev-cpp — APPROVED** [review]
LGTM!

**Review by alaibe — APPROVED** [review]

File: src/app/modules/main/wallet_section/collectibles/models/collectibles_model.nim [hunk 1]
```diff
@@ -167,44 +182,53 @@ QtObject:
...
-        self.items.delete(i, i + loadingItemsCount - 1)
-        self.endRemoveRows()
-        self.countChanged()
-        return
+    self.beginRemoveRows(parentModelIndex, self.loadingItemsStartIdx, self.loadingItemsStartIdx + loadingItemsCount - 1)
+    self.items.delete(self.loadingItemsStartIdx, self.loadingItemsStartIdx + loadingItemsCount - 1)
+    self.loadingItemsStartIdx = -1
+    self.endRemoveRows()
+    self.countChanged()
+    return
```
Comment (saledjenic) [inline src/app/modules/main/wallet_section/collectibles/models/collectibles_model.nim:212]: you don't need return here

File: src/app_service/service/collectible/service.nim [hunk 2]
```diff
@@ -49,43 +51,79 @@ proc `$`*(self: OwnedCollectible): string =
...
+    of State.WatchedContractsLoading, State.ChunkLoading:
+      true
+    else:
+      false
+
+proc anyLoaded*(self: CollectiblesData): bool =
+  return self.lastRefetchTimestamp != INVALID_TIMESTAMP.utc()
+
+proc isError*(self: CollectiblesData): bool =
+  result = case self.state:
```
Comment (saledjenic) [inline src/app_service/service/collectible/service.nim:104]: `return self.state == State.Error`

File: src/app_service/service/collectible/service.nim [hunk 3]
```diff
@@ -49,43 +51,79 @@ proc `$`*(self: OwnedCollectible): string =
...
+    of State.Error:
+      true
+    else:
+      false
+
+proc allLoaded*(self: CollectiblesData): bool =
+  result = case self.state:
```
Comment (saledjenic) [inline src/app_service/service/collectible/service.nim:111]: `return self.state == State.AllLoaded`

**Review by saledjenic — APPROVED** [review]

**Pull request merged**